react/src/services/auth.js

```diff
@@ -5,7 +5,7 @@ export const postLogin = (email, password) => api.post(
   {email: email, password: password}
 );
 
-export const postSignUp = (email, password, confirm_password, first_name, last_name) => api.post(
+export const postSignUp = (email, password, password_confirmation, first_name, last_name) => api.post(
```

Comment:
Linter usually yields at this point, by reclaiming that these should be written in `camelCase`. Here I'd let them as they are.